featurebyte/api/target_namespace.py

```diff
@@ -0,0 +1,145 @@
+"""
+Feature Namespace module.
+"""
+from __future__ import annotations
+
+from typing import List, Optional
+
+from pydantic import Field
+
+from featurebyte.api.api_handler.base import ListHandler
+from featurebyte.api.api_handler.target_namespace import TargetNamespaceListHandler
+from featurebyte.api.api_object_util import ForeignKeyMapping
+from featurebyte.api.base_table import TableApiObject
+from featurebyte.api.entity import Entity
+from featurebyte.api.savable_api_object import SavableApiObject
+from featurebyte.exception import RecordRetrievalException
+from featurebyte.models.base import PydanticObjectId
+from featurebyte.models.feature_namespace import DefaultVersionMode
+from featurebyte.models.target_namespace import TargetNamespaceModel
+from featurebyte.schema.target_namespace import TargetNamespaceUpdate
+
+
+class TargetNamespace(SavableApiObject):
```

Comment:
Should we refactor this & `FeatureNamespace`? Since both of them share many similarities.